src/ConcreteOperations/isdisjoint.jl

```diff
@@ -1573,3 +1573,44 @@ end
 function is_intersection_empty(::Line2D{N}, ::EmptySet{N}) where {N<:Real}
     return true
 end
+
+# =================================================================
+# Disjointness methods for two-dimensional lines and line segments
+# =================================================================
+
+"""
+    is_intersection_empty(L1::Line2D{N}, L2::Line2D{N}, witness::Bool=false) where {N<:Real}
+
+Check whether two two-dimensional lines do not intersect.
+
+### Input
+
+- `L1` -- line
+- `L2` -- line
+
+### Output
+
+* If `witness` option is deactivated: `true` iff ``L1 ∩ L2 = ∅``
+* If `witness` option is activated:
+  * `(true, [])` iff ``L1 ∩ L2 = ∅``
+  * `(false, v)` iff ``L1 ∩ L2 ≠ ∅`` and ``v ∈ L1 ∩ L2``
+"""
+function is_intersection_empty(L1::Line2D{N}, L2::Line2D{N}, witness::Bool=false) where {N<:Real}
+    disjoint, _ = _isdisjoint(L1, L2)
+    if !witness
+        return disjoint
+    else
+        if disjoint
+            return (true, [])
+        else
+            return (false, an_element(intersection(L1, L2)))
```

Comment:
Apparently we do not have `an_element` for `Line2D` and the intersection can be of that type.
```
LoadError: MethodError: no method matching element(::Line2D{Float64,SingleEntryVector{Float64}})
```

Reply:
```julia
julia> L = rand(Line2D)
Line2D{Float64,Array{Float64,1}}([-0.6939299436010502, 0.856395234301458], 0.6582110708021547)

julia> an_element(L)
2-element Array{Float64,1}:
 0.28559678873468836
 1.0
```

Reply:
Oh, sorry, I misread that error message.

```julia
  LoadError: MethodError: no method matching element(::Line2D{Float64,SingleEntryVector{Float64}})
```

Reply:
The error can be reproduced with:

```julia
julia> N = Float64; H1 = Hyperrectangle(N[1, 1], N[2, 2])
Hyperrectangle{Float64,Array{Float64,1},Array{Float64,1}}([1.0, 1.0], [2.0, 2.0])

julia> vlist = vertices_list(H1)
4-element Array{Array{Float64,1},1}:

 [3.0, 3.0]
 [-1.0, 3.0]
 [3.0, -1.0]
 [-1.0, -1.0]

julia> H1pol = convert(HPolygon, H1)
HPolygon{Float64,LazySets.Arrays.SingleEntryVector{Float64}}(HalfSpace{Float64,LazySets.Arrays.SingleEntryVector{Float64}}[HalfSpace{Float64,LazySets.Arrays.Si
ngleEntryVector{Float64}}([1.0, 0.0], 3.0), HalfSpace{Float64,LazySets.Arrays.SingleEntryVector{Float64}}([0.0, 1.0], 3.0), HalfSpace{Float64,LazySets.Arrays.S
ingleEntryVector{Float64}}([-1.0, 0.0], 1.0), HalfSpace{Float64,LazySets.Arrays.SingleEntryVector{Float64}}([0.0, -1.0], 1.0)])

julia> vlist = vertices_list(H1pol)
ERROR: MethodError: no method matching element(::Line2D{Float64,LazySets.Arrays.SingleEntryVector{Float64}})
Closest candidates are:
  element(::Singleton{N,VN} where VN<:AbstractArray{N,1}) where N<:Real at /home/mforets/.julia/dev/LazySets/src/Sets/Singleton.jl:37
  element(::Singleton{N,VN} where VN<:AbstractArray{N,1}, ::Int64) where N<:Real at /home/mforets/.julia/dev/LazySets/src/Sets/Singleton.jl:55
  element(::ZeroSet{N}) where N<:Real at /home/mforets/.julia/dev/LazySets/src/Sets/ZeroSet.jl:43
  ...
Stacktrace:
 [1] vertices_list(::HPolygon{Float64,LazySets.Arrays.SingleEntryVector{Float64}}; apply_convex_hull::Bool, check_feasibility::Bool) at /home/mforets/.julia/de
v/LazySets/src/Interfaces/AbstractHPolygon.jl:150
 [2] vertices_list(::HPolygon{Float64,LazySets.Arrays.SingleEntryVector{Float64}}) at /home/mforets/.julia/dev/LazySets/src/Interfaces/AbstractHPolygon.jl:140
 [3] top-level scope at REPL[14]:1
```

so at some point the `vertices_list` function expects a singleton but receives a line.

Reply:
The last commits fixes this error.